src/connectors/userService.ts

```diff
@@ -434,6 +440,7 @@ export class UserService extends BaseService {
     if (!user.email || !user.emailVerified) {
       throw new ForbiddenError('email not verified')
     }
+    auditLog({ actorId: user.id, action: 'update_password' })
```

Comment:
For security and product analysis reasons, I believe we should not only log the successful result of an action but also the attempt and failure result:

```
auditLog({ actorId: user.id, action: 'update_password', status: 'pending' })
try {
  ...
  auditLog({ actorId: user.id, action: 'update_password', status: 'successed' })
} catch (error) {
  auditLog({ actorId: user.id, action: 'update_password', status: 'failed' })
  ...
}
```